back/spec/models/permission_spec.rb

```diff
@@ -44,7 +44,7 @@
     let!(:permission_voting) { create(:permission, action: 'voting_idea', permission_scope: project) }
 
     it 'Returns permissions in the correct order' do
-      permissions = described_class.all.order_by_action
+      permissions = described_class.all.order_by_action(project)
```

Comment:
Is `.all` necessary here?

Reply:
No, indeed it works perfectly without it!